plugins/API/templates/rateLimit.twig

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+    <meta charset="UTF-8">
+    <title>Matomo Cloud - 429 Too Many Requests</title>
```

Comment:
We shouldn't add a file like this to core. It's cloud specific and not useful for any on premise install.